VERSIONS.md

```diff
@@ -1,13 +1,13 @@
+| 7.6.1 | [5.19.0](https://github.com/RevenueCat/purchases-ios/releases/tag/5.19.0) | [8.13.0](https://github.com/RevenueCat/purchases-android/releases/tag/8.13.0) | [13.24.0](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/13.24.0) | [7.1.1](https://developer.android.com/google/play/billing/release-notes) | Unity IAP 5.0.0-pre.4+ |
+| 7.6.0 | [5.18.0](https://github.com/RevenueCat/purchases-ios/releases/tag/5.18.0) | [8.13.0](https://github.com/RevenueCat/purchases-android/releases/tag/8.13.0) | [13.22.0](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/13.22.0) | [7.1.1](https://developer.android.com/google/play/billing/release-notes) | Unity IAP 5.0.0-pre.4+ |
+| 7.5.3 | [5.18.0](https://github.com/RevenueCat/purchases-ios/releases/tag/5.18.0) | [8.12.2](https://github.com/RevenueCat/purchases-android/releases/tag/8.12.2) | [13.19.0](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/13.19.0) | [7.1.1](https://developer.android.com/google/play/billing/release-notes) | Unity IAP 5.0.0-pre.4+ |
+| 7.5.2 | [5.16.1](https://github.com/RevenueCat/purchases-ios/releases/tag/5.16.1) | [8.11.0](https://github.com/RevenueCat/purchases-android/releases/tag/8.11.0) | [13.17.1](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/13.17.1) | [7.1.1](https://developer.android.com/google/play/billing/release-notes) | Unity IAP 5.0.0-pre.4+ |
+| 7.5.1 | [5.15.0](https://github.com/RevenueCat/purchases-ios/releases/tag/5.15.0) | [8.11.0](https://github.com/RevenueCat/purchases-android/releases/tag/8.11.0) | [13.16.2](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/13.16.2) | [7.1.1](https://developer.android.com/google/play/billing/release-notes) | Unity IAP 5.0.0-pre.4+ |
+| 7.5.0 | [5.15.0](https://github.com/RevenueCat/purchases-ios/releases/tag/5.15.0) | [8.11.0](https://github.com/RevenueCat/purchases-android/releases/tag/8.11.0) | [13.16.0](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/13.16.0) | [7.1.1](https://developer.android.com/google/play/billing/release-notes) | Unity IAP 5.0.0-pre.4+ |
+| 7.4.2 | [5.14.5](https://github.com/RevenueCat/purchases-ios/releases/tag/5.14.5) | [8.10.8](https://github.com/RevenueCat/purchases-android/releases/tag/8.10.8) | [13.15.2](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/13.15.2) | [7.1.1](https://developer.android.com/google/play/billing/release-notes) | Unity IAP 5.0.0-pre.4+ |
 | 7.4.1 | [5.14.0](https://github.com/RevenueCat/purchases-ios/releases/tag/5.14.0) | [8.10.5](https://github.com/RevenueCat/purchases-android/releases/tag/8.10.5) | [13.14.0](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/13.14.0) | [7.0.0](https://developer.android.com/google/play/billing/release-notes) | Not compatible |
```

Comment:
Note I didn't update the ones using BC 7.0.0... It would probably be ok, but there might be some incompatibility, so I think it's better to be safe and just ask people to use a higher version